packages/eas-cli/src/build/utils/url.ts

```diff
@@ -10,7 +10,9 @@ export function getProjectDashboardUrl(accountName: string, projectName: string)
 export function getBuildLogsUrl(build: BuildFragment): string {
   const { project } = build;
   if (project.__typename === 'App') {
-    return `${getExpoWebsiteBaseUrl()}/accounts/${project.ownerAccount.name}/builds/${build.id}`;
+    return `${getExpoWebsiteBaseUrl()}/accounts/${project.ownerAccount.name}/projects/${
+      project.slug
```

Comment:
Is this value safe? Or does this need to be url encoded?

Reply:
The project's slug is URL safe because we are already used it in the URL like this: `https://expo.dev/@community/native-component-list`.

For reference: https://github.com/expo/eas-cli/pull/845

Reply:
Perfect, thanks for confirming.

Reply:
@kbrandwijk Can you approve CI for testing. This is my first time contribut on this repo.